spec/support/shared_contexts/aruba.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.shared_context 'uses aruba API' do
+  def random_string(options = {})
+    options[:prefix].to_s + SecureRandom.hex + options[:suffix].to_s
+  end
+
+  before(:each) do
+    klass = Class.new {
+      include Aruba::Api
+
+      def set_tag(tag_name, value)
+        self.instance_variable_set "@#{tag_name}", value
+      end
+
+      def announce_or_puts(*args)
+        p caller[0..2]
+      end
+    }
```

Comment:
This feels a bit messy. Is it necessary to define a new instance of this `klass` per example? Could you not just define an `ArubaRunner` class or somesuch once, then use instances of that for `@aruba`?